week_1/Project/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,123 @@
+.header {
+    height: 55px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 100;
+}
+
+.left-section {
+    display: flex;
+    align-items: center;
+}
+
+.hamburger-menu {
+    height: 24px;
+    margin: 0px 20px 0px 24px;
+}
+
+.youtube-logo {
+    height: 22px;
+}
+
+.middle-section {
+    flex: 1;
+    margin: 0px 35px 0px 70px;
+    max-width: 650px;
+    display: flex;
+    align-items: center;
+}
+
+.search-bar {
+    flex: 1;
+    height: 34px;
+    padding-left: 10px;
+    font-size: 16px;
+    border-width: 1px;
+    border-style: solid;
+    border-top-left-radius: 20px;
+    border-bottom-left-radius: 20px;
```

Comment:
@codal-kcolaco please check how we can give all border attributes in one line...

Reply:
@abhi-codal  Yes will check for sure.